crates/aptos/src/common/init.rs

```diff
@@ -202,7 +202,7 @@ impl CliCommand<()> for InitTool {
                 let input = input.trim();
                 if input.is_empty() {
                     if let Some(key) = profile_config.private_key {
-                        eprintln!("No key given, keeping existing key...");
+                        eprintln!("Private key given, keeping existing key...");
```

Comment:
Maybe this message could be more clear, though I'm not sure this is the right one.

So, let's brainstorm the situation:
1. There's a private key that already exists
2. The user has not given a private key.

So, I think this would be:
```
No private key given, keeping existing private key...
```